core-api/src/main/java/com/optimizely/ab/bucketing/DecisionService.java

```diff
@@ -81,24 +83,28 @@ public DecisionService(@Nonnull Bucketer bucketer,
      * @param experiment         The Experiment the user will be bucketed into.
      * @param userId             The userId of the user.
      * @param filteredAttributes The user's attributes. This should be filtered to just attributes in the Datafile.
+     * @param projectConfig      The current projectConfig
+     * @param options            An array of decision options
+     * @param reasons            Decision log messages
      * @return The {@link Variation} the user is allocated into.
      */
     @Nullable
     public Variation getVariation(@Nonnull Experiment experiment,
                                   @Nonnull String userId,
                                   @Nonnull Map<String, ?> filteredAttributes,
-                                  @Nonnull ProjectConfig projectConfig) {
-
-        if (!ExperimentUtils.isExperimentActive(experiment)) {
+                                  @Nonnull ProjectConfig projectConfig,
+                                  @Nonnull List<OptimizelyDecideOption> options,
+                                  @Nonnull DecisionReasons reasons) {
+        if (!ExperimentUtils.isExperimentActive(experiment, options, reasons)) {
```

Comment:
options not needed.

Reply:
Not used for now, but will keep this consistently for future options.